x-pack/plugins/rule_registry/server/search_strategy/search_strategy.ts

```diff
@@ -91,18 +91,31 @@ export const ruleRegistrySearchStrategyProvider = (
             filter.push(getSpacesFilter(space.id) as estypes.QueryDslQueryContainer);
           }
 
+          const sort = request.sort
+            ? request.sort.map(({ field, direction }) => {
+                return {
+                  [field]: {
+                    order: direction,
+                  },
+                };
+              })
+            : {};
```

Comment:
super small nitpick: should we try to be consistent with the types? (array)

Reply:
Sure